include/SDL3/SDL_stdinc.h

```diff
@@ -1291,6 +1291,22 @@ extern SDL_DECLSPEC int SDLCALL SDL_strcasecmp(const char *str1, const char *str
  */
 extern SDL_DECLSPEC int SDLCALL SDL_strncasecmp(const char *str1, const char *str2, size_t maxlen);
 
+
+
+/**
+ * Locate the first occurrence of any character in the accept string.
+ *
+ * \param s String in which characters a look for characters. MULL is not permitted!
```

Comment:
MULL -> NULL

Reply:
Also, "String in which characters a look for characters" seems to have either extra words or missing words, depending on the intended wording.